Add async-send changes to release note (#7516)
ballerina-platform/ballerina-dev-website: downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md

### Improvements

> #### Make `async-send` an action node

Reviewer: Should be say "node"? 
```suggestion
#### Make `async-send` an action
```

> #### Make `async-send` as an action node

Reviewer 2: ```suggestion
#### Make `async-send` an action node
```

> #### Make `async-send` as an action node

Reviewer: ```suggestion
#### Make `async-send` an action node
```

> `async-send` has been reclassified from a statement to an action, in alignment with the Ballerina specification.

Reviewer 2: ```suggestion
- Reclassified `async-send` from a statement to an action in alignment with the Ballerina specification.
```

> `async-send` has been reclassified from a statement to an action node, aligning with the Ballerina specification.

Reviewer 2: ```suggestion
`async-send` has been reclassified from a statement to an action node aligning with the Ballerina specification.
```

> `async-send` has been reclassified from a statement to an action node, aligning with the Ballerina specification.

Reviewer: ```suggestion
`async-send` has been reclassified from a statement to an action, in alignment with the Ballerina specification.
```
